### ISSUE TITLE
Line / page segmentation

### ISSUE THREAD

**Issue by TsvikaKuflik** [ajwecker/TS1218#35]
The page / line segemnter cuts the end of a letter:
![image](https://user-images.githubusercontent.com/47208723/52521849-584a1280-2c86-11e9-8f6b-7d2585beeb7a.png)

**Comment by dstoekl** [comment]
yes. it comes from the difficult binarization. Uri, can you simply deduce 50 pixels from the x1 column?

**Comment by dstoekl** [comment]
I have added a new file Geneva_146_all_data_for_Alan05.txt with 150 pixels more to the left to the gdrive. Can you upload it to the database?

**Comment by dstoekl** [comment]
@ajwecker 
have you updated the linesegmentation in the new version?

**Comment by ajwecker** [comment]
no
